main.go

```diff
@@ -92,13 +198,30 @@ func store_services(systemKey string, services []*cb.Service, meta *System_meta)
 	}
 
 	for _, service := range services {
-		if err := ioutil.WriteFile(dir+"/"+service.Name+".js", []byte(service.Code), 0777); err != nil {
+		if err := ioutil.WriteFile(codedir+"/"+service.Name+".js", []byte(service.Code), 0777); err != nil {
```

Comment:
This will cause the tool to break for those who were using the old version right? @sh3nan1gans